mscore/musescore.cpp

```diff
@@ -1,4 +1,4 @@
-//=============================================================================
+﻿//=============================================================================
```

Comment:
I'm not sure how to fix this. I tried deleting the line and pasting it again, but no change is detected.